code/modules/reagents/chemistry/reagents_datum.dm

```diff
@@ -107,6 +109,16 @@
 	holder.remove_reagent(id, total_depletion_rate) //By default it slowly disappears.
 	return STATUS_UPDATE_NONE
 
+/datum/reagent/proc/on_mob_od_life(mob/living/M) //We want to drain reagents but not do the entire mob life.
```

Comment:
can you renamed this proc to not include shortened versions of words like `od`, it may not be immediately clear to coders down the line that `od` -> `overdose`

Reply:
Done!